packages/code-du-travail-frontend/src/lib/Link.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import PropTypes from "prop-types";
+import NextLink from "next/link";
+
+const Link = ({ pathname, query, alias, slugKey, children, ...props }) => {
```

Comment:
I think pathname should be named `route` or `page`  since its what we want to link to.

I think we should simplify the slug / slugKey before to avoid address edge case here
also I would add a comment about how nextLink work when we want clean urls and use route masking